core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -366,6 +373,19 @@ public FeaturesConfig setExchangeCompressionCodec(CompressionCodec exchangeCompr
         return this;
     }
 
+    @Config("experimental.blockserde-vectorized-null-suppression-strategy")
```

Comment:
`exchange.experimental.vectorized-serde.enabled`
blockserde...null-suppression implies a very specific thing, we may want to use Vector API for more than just that
having a more generic name keeps the door open to adding more usage under same flag and is easier for users to grasp

Reply:
Sure, will change the value